BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.m

```diff
@@ -212,6 +212,7 @@ - (void)finish
 
 - (void)dealloc
 {
+    self.receivedDataBuffer = nil;
```

Comment:
This isn't harmful, but should be unnecessary. Did this appear to fix something?

Reply:
As I noted... I could never repro the exact issue.

Reply:
James and I chatted offline. And I shared my sketchy theories.  James agreed that it that scenario this might help (if some other code is doing something unsafe). So this change wouldn't fix *that* problem, but could mitigate the outcome by avodiing a crash here.